adal/src/main/java/com/microsoft/aad/adal/HashMapExtensions.java

```diff
@@ -109,11 +109,11 @@ private HashMapExtensions() {
      * get key value pairs from response.
      *
      * @param webResponse HttpWebResponse to convert to a map
-     * @return Map
+     * @return HashMap
      * @throws JSONException
      */
-    static Map<String, String> getJsonResponse(HttpWebResponse webResponse) throws JSONException {
-        final Map<String, String> response = new HashMap<>();
+    static HashMap<String, String> getJsonResponse(HttpWebResponse webResponse) throws JSONException {
```

Comment:
I'm assuming this is using `HashMap` because it implements `Serializable` and this is going to be on an `Intent` -- where is this happening though?

Reply:
in BrokerProxy, please check the code here. https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/dev/adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java#L485